cli/tests/pcluster/validators/test_all_validators.py

```diff
@@ -24,12 +25,25 @@
     kms_validators,
     networking_validators,
     s3_validators,
+    scheduler_plugin_validators,
 )
 from pcluster.validators.common import Validator
 from tests.pcluster.aws.dummy_aws_api import mock_aws_api
 
 
-def _mock_all_validators(mocker, mockers):
+def _mock_all_validators(mocker, mockers, additional_modules=None):
+    modules = [
+        cluster_validators,
+        ebs_validators,
+        ec2_validators,
+        fsx_validators,
+        kms_validators,
+        iam_validators,
+        networking_validators,
+        s3_validators,
+    ]
+    if additional_modules:
+        modules += additional_modules
     for module in [
         cluster_validators,
         ebs_validators,
```

Comment:
Shouldn't we be looping over `modules` now instead of using an explicit list here? Or am I missing something?

Reply:
Good catch, replaced with modules